product_sticker/tests/test_product.py

```diff
@@ -2,6 +2,23 @@
 
 
 class TestStickersOnProducts(ProductStickerCommon):
+    def _test_model_availability(self, product):
+        random_models = self.env["ir.model"].search([], limit=2)
```

Comment:
thought: I understand that this *should not fail*. However, it opens the door for uncontrolled inputs, and thus non-reproducible tests... Couldn't you just use some pre-controlled set of models? Just to make sure tests are reproducible. For example:

```suggestion
        random_models = self.env.ref("base.model_ir_model") + self.env.ref("base.model_ir_ui_view")
```